news/2018-10-10-beatmap-spotlights-summer-2018.md

```diff
@@ -0,0 +1,311 @@
+---
+layout: post
+title: "Beatmap Spotlights: Summer 2018"
+date: 2018-10-10 06:00:00 +0000
+---
+
+Welcome back to a long-awaited rendition of this season's beatmap spotlights! We have lots of brilliant maps to showcase and some extra surprises for all of you!
+
+![](/wiki/shared/news/banners/spotlights-header.jpg)
+
+The beatmap charts for the Summer Spotlights can be found [here](https://osu.ppy.sh/p/chart?ch=SUMMER2018). Their respective packs are available at the [Beatmap Pack Listing](https://osu.ppy.sh/p/packlist?t=r) for easier access. **The Summer 2018 Spotlights rankings will end on the 31st of December, 2018.**
```

Comment:
The rankings should not end on the same day as the release of the next due to the announcement of the winners. Might need to move that date a bit back, to December 15 for example.